umap/templates/umap/map_table.html

```diff
@@ -0,0 +1,29 @@
+{% load umap_tags umap_tags i18n %}
+<table class="maps">
+  <tr>
+    <th></th>
+    <th></th>
+    <th>{% blocktrans %}Status{% endblocktrans %}</th>
+    <th>{% blocktrans %}Last save{% endblocktrans %}</th>
+    <th>{% blocktrans %}Owner{% endblocktrans %}</th>
+    <th>{% blocktrans %}Actions{% endblocktrans %}</th>
+  </tr>
+  {% for map_inst in maps %}
```

Comment:
That part should be in a `<tbody>`

Reply:
Can you elaborate on why we should add those tags ? I thought that HTML5 thing was to only add needed tags, to KISS :)

Reply:
Added `<thead>` and `<tbody>`, but not sure why we'd have a `<caption>`.

Note that all seems optional according to (my understanding of) MDN: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table :)